crm_project_task/tests/common.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+from odoo.tests.common import TransactionCase
+
+
+class CrmProjectTaskCommon(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
This method is small and we don't expect many modules to extend this one, so you can just put the method in `TestCrmProjectTask` directly if you prefer. Not blocking.